README.md

```diff
@@ -53,13 +53,13 @@ Versions of supported components
 --------------------------------
 
 
-[kubernetes](https://github.com/kubernetes/kubernetes/releases) v1.6.7 <br>
-[etcd](https://github.com/coreos/etcd/releases) v3.0.17 <br>
+[kubernetes](https://github.com/kubernetes/kubernetes/releases) v1.7.3 <br>
+[etcd](https://github.com/coreos/etcd/releases) v3.2.4 <br>
 [flanneld](https://github.com/coreos/flannel/releases) v0.8.0 <br>
-[calicoctl](https://github.com/projectcalico/calico-docker/releases) v0.23.0 <br>
+[calicoctl](https://github.com/projectcalico/calico-docker/releases) v1.1.3 <br>
```

Comment:
not related to your change, but im curious if we should instead change this to reflect calico's versioning ( I think after the most recent calico change, we're somewhere around v2.1.5 ( https://docs.projectcalico.org/v2.1/releases/#v215 )